### PR TITLE
Missing Tokens: ~800 mainnet 16 gnosis tokens

### PR DESCRIPTION
**Pull request by bh2smith**
Just like the old days (using the script from [this repo](https://github.com/cowprotocol/data-misc)). Although, I had to make some changes to the script (update dune-client and web3 versions).

### CODE REVIEW

File: tokens/models/tokens/tokens_erc20.sql [hunk 1]
```diff
@@ -6,7 +6,7 @@
         ,post_hook='{{ expose_spells(\'["arbitrum","avalanche_c","base","bnb","celo","ethereum","fantom","gnosis","goerli","optimism","polygon","scroll","zksync","zora"]\',
                         "sector",
                         "tokens",
-                        \'["0xManny","hildobby","soispoke","dot2dotseurat","mtitus6","wuligy","lgingerich","0xRob","jeff-dude"]\') }}'
+                        \'["bh2smith", "0xManny","hildobby","soispoke","dot2dotseurat","mtitus6","wuligy","lgingerich","0xRob","jeff-dude"]\') }}'
```
Comment (bh2smith) [inline tokens/models/tokens/tokens_erc20.sql:9]: This should have already been here.

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {{
     config(
-        schema = 'tokens_ethereum'
-        ,alias = 'erc20'
-        ,tags = ['static']
-        ,materialized = 'table'
+    schema = 'tokens_ethereum'
+    ,alias = 'erc20'
+    ,tags = ['static']
+    ,materialized = 'table'
```
Comment (bh2smith) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:6]:
```suggestion
        schema = 'tokens_ethereum'
        ,alias = 'erc20'
        ,tags = ['static']
        ,materialized = 'table'

```

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 3]
```diff
@@ -4,6 +4,7 @@
         ,alias = 'erc20'
         ,tags = ['static']
         ,materialized = 'table'
+
```
Comment (bh2smith) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:7]:
```suggestion
```

File: tokens/models/tokens/gnosis/tokens_gnosis_erc20.sql [hunk 4]
```diff
@@ -7775,4 +7775,20 @@ FROM (VALUES
...
+    , ( 0x0c1b9ce6bf6c01f587c2ee98b0ef4b20c6648753, 'EURe/staBAL3', 18)
+    , ( 0x861d6dbd673694478ad0cd28be45073f60da9494, 'ysyvAjnaEURe', 18)
+    , ( 0x0ca1c1ec4ebf3cc67a9f545ff90a3795b318ca4a, 'crvEUReUSD', 18)
+    , ( 0xc9f00c3a713008ddf69b768d90d4978549bfdf94, 'crvUSD/sDAI', 18)
+    , ( 0x4683e340a8049261057d5ab1b29c8d840e75695e, 'B-50wstETH-50GNO', 18)
+    , ( 0xbc2acf5e821c5c9f8667a36bb1131dad26ed64f9, 'B-50sDAI-50wstETH', 18)
+    , ( 0x5f603c60adb40691939b9a740beecef4b0f5c146, 'mooAuraGnosisWstETH-GNO', 18)
+    , ( 0x5012c6bf79b3047ecfff2f212dffda4d2128188f, 'yvEURe-1', 18)
+    , ( 0x7644fa5d0ea14fcf3e813fdf93ca9544f8567655, 'sBAL3', 18)
 ) AS temp_table (contract_address, symbol, decimals)
```
Comment (bh2smith) [inline tokens/models/tokens/gnosis/tokens_gnosis_erc20.sql:7794]:
```suggestion
) AS temp_table (contract_address, symbol, decimals)

```

**Comment by bh2smith** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**